src/main/java/com/demo/MybatisApplication/exceptions/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,19 @@
+package com.demo.MybatisApplication.exceptions;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.Date;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+    @ExceptionHandler(ResourceNotFoundException.class)
+    public ResponseEntity<?> studentNOtFoundException(ResourceNotFoundException ex, WebRequest request){
```

Comment:
```suggestion
    public ResponseEntity<?> resourceNotFoundException(ResourceNotFoundException ex, WebRequest request){
```

ResourceNotFoundEx.. is a generic exception and can be thrown when an entity is not found. It will not be just used for Student entity so we should specify a generic name.

Reply:
Done